modules/monitoring-about-specifying-limits-and-requests-for-monitoring-components.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-specifying-limits-and-requests-for-monitoring-components_{context}"]
+= About specifying limits and requests for monitoring components
+
+Specify limits and requests to manage CPU and memory resource consumption for the containers that run monitoring components. 
+Doing so ensures that the containers in the pods have enough resources to run smoothly.
+
+When you define limits for a container, you specify a cap on a container's resource usage, preventing the container from exceeding the specified values for CPU and memory resources.
+
+When you define requests for a container, you specify that a container should only be scheduled on a node with enough resources to fulfill that request.
+The specified amounts of CPU and memory resources are guaranteed to be available for the container.
+
+You can configure settings for all monitoring components, including the following:
+
+* Alertmanager
+* kube-state-metrics
+* monitoring-plugin
+* node-exporter
+* openshift-state-metrics
+* Prometheus
```

Comment:
in-cluster prometheus and UWM prometheus both are support resources.requests and resources.limits
in-cluster prometheus: https://github.com/openshift/cluster-monitoring-operator/blob/release-4.15/pkg/manifests/types.go#L205
UWM prometheus: https://github.com/openshift/cluster-monitoring-operator/blob/release-4.15/pkg/manifests/types.go#L643

Reply:
Updated the list item to make this clear.